moveit_core/online_signal_smoothing/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Plugin implementations
+
+add_library(moveit_butterworth_filter SHARED
+  src/butterworth_filter.cpp
+)
+include(GenerateExportHeader)
+generate_export_header(moveit_butterworth_filter)
+target_include_directories(moveit_butterworth_filter PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+)
+set_target_properties(moveit_butterworth_filter PROPERTIES VERSION
+  "${${PROJECT_NAME}_VERSION}"
+)
+target_link_libraries(moveit_butterworth_filter
+  moveit_robot_model
+)
+
+# Installation
+
+install(DIRECTORY include/ DESTINATION include)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/moveit_butterworth_filter_export.h DESTINATION include)
+install(TARGETS moveit_butterworth_filter EXPORT moveit_butterworth_filter)
```

Comment:
No need to install the targets here. The top-level CMakeLists.txt in `moveit_core` already takes care of that.